examples/tick-single-cluster/main.tf

```diff
@@ -0,0 +1,221 @@
+  source = "../../modules/chronograf-security-group-rules"
+
+  security_group_id = "${module.tick.security_group_id}"
+
+  http_port = 8888
+
+  # To keep this example simple, we allow these ports to be accessed from any IP. In a production
+  # deployment, you may want to lock these down just to trusted servers.
+  http_port_cidr_blocks = ["0.0.0.0/0"]
+}
+
+# ---------------------------------------------------------------------------------------------------------------------
+# ATTACH IAM POLICIES TO EACH CLUSTER
+# These policies allow the clusters to automatically bootstrap themselves
+# ---------------------------------------------------------------------------------------------------------------------
+
+module "influxdb_iam_policies" {
+  # When using these modules in your own code, you will need to use a Git URL with a ref attribute that pins you
+  # to a specific version of the modules, such as the following example:
+  # source = "git::git@github.com:gruntwork-io/terraform-aws-influx.git//modules/influxdb-iam-policies?ref=v0.0.1"
+  source = "../../modules/influxdb-iam-policies"
+
+  iam_role_id = "${module.tick.iam_role_id}"
+}
+
+# ---------------------------------------------------------------------------------------------------------------------
+# DEPLOY A LOAD BALANCER FOR THE CLUSTERS
+# ---------------------------------------------------------------------------------------------------------------------
+
+module "load_balancer" {
+  # When using these modules in your own code, you will need to use a Git URL with a ref attribute that pins you
+  # to a specific version of the modules, such as the following example:
+  # source = "git::git@github.com:gruntwork-io/terraform-aws-influx.git//modules/load-balancer?ref=v0.0.1"
+  source = "../../modules/load-balancer"
+
+  name       = "${var.cluster_name}-lb"
+  vpc_id     = "${data.aws_vpc.default.id}"
+  subnet_ids = "${data.aws_subnet_ids.default.ids}"
+
+  http_listener_ports = [8086, 8888]
+
+  # To make testing easier, we allow inbound connections from any IP. In production usage, you may want to only allow
+  # connectsion from certain trusted servers, or even use an internal load balancer, so it's only accessible from
+  # within the VPC
+
+  allow_inbound_from_cidr_blocks = ["0.0.0.0/0"]
+  idle_timeout                   = 3600
+}
+
+module "influxdb_target_group" {
+  # When using these modules in your own code, you will need to use a Git URL with a ref attribute that pins you
+  # to a specific version of the modules, such as the following example:
+  # source = "git::git@github.com:gruntwork-io/terraform-aws-influx.git//modules/load-balancer-target-group?ref=v0.0.1"
+  source = "../../modules/load-balancer-target-group"
+
+  target_group_name    = "${var.cluster_name}-itg"
+  asg_name             = "${module.tick.asg_name}"
+  port                 = "${module.influxdb_security_group_rules.api_port}"
+  health_check_path    = "/ping"
+  health_check_matcher = "204"
```

Comment:
Why 204? Might want to add link to relevant docs.

Reply:
InfluxDB specifically has a ping endpoint for checking its health. I'll link to it